lib/storage/file.js

```diff
@@ -798,6 +798,68 @@ File.prototype.setMetadata = function(metadata, callback) {
 };
 
 /**
+ * Set a file to be private to the project and removes all other permissions.
+ * If you set `options.strict` to true, it will set it to be private to the
+ * owner user only.
+ *
+ * @param {object=} options - The configuration object.
+ * @param {object=} options.strict - If true, set the file to be private to only
+ *     the owner user. Otherwise, it will be private to the project.
+ * @param {function=} callback - The callback function.
+ *
+ * @example
+ *
+ * //-
+ * // Set the file private so only project maintainers can see and modify it.
+ * //-
+ * file.makePrivate(function(err) {});
+ *
+ * //-
+ * // Set the file private so only owner can see and modify it.
+ * //-
+ * file.makePrivate({ strict: true }, function(err) {});
+ */
+File.prototype.makePrivate = function(options, callback) {
+  var that = this;
+  if(util.is(options, 'function')) {
+      callback = options;
+      options = {};
+  }
+  var path = '/o/' + encodeURIComponent(this.name);
+  var query = { predefinedAcl: (options.strict ? '' : 'project') + 'private' };
```

Comment:
In [the docs](https://cloud.google.com/storage/docs/json_api/v1/objects/patch), it asks for `projectPrivate` - does the casing matter?